### DISCUSSION TITLE
Support for Analogue / RetroFreak adapters

### DISCUSSION THREAD

**Discussion by toaks2804** [sanni/cartreader#693] [Hardware]
Hello. I have various cart adapters from Analogue that I am hoping are compatible with the Sanni. There is one for the Master System, Game Gear, SG-1000 Carts and Cards and they connect to the Genesis port. They also have a Game Gear, Neo Geo Pocket, TG-16, and Atari Lynx that connect to the GBA port. Has anyone tried them? Also I have a cart adapter from RetroFreak for SG-1000, Game Gear and Sega Card games. Anyone tried this one? If they dont work, could support be added for them?

**Comment by PsyK0p4T** [comment]
Can't say for the Analogue ones. But if the adapters' pins are more/less numerous, or with a different mapping than the current ones supported by OSCR, then they won't work as-it...

About the RetroFreak SEGA adapter you have, is it different than "Retron5 3-in-1" ?
Just googled for some pics and they look quite similar from the outside. Can't say for the inside.
You can maybe give it a try with the current code + Game Gear carts, by using Retron mode, but no guarantee...

**Reply by toaks2804** [reply]
I think the RetroFreak and Retron5 adapters are pretty much the same. I'm pretty sure I remember reading somewhere that the Retron5 adapter worked on the Retro Freak console.

**Comment by PsyK0p4T** [comment]
Months later, I can finally answer to this !

So yes, "RetroFreak RF-5" and "Retron5 3-in-1" adapters are quite close.
A difference between them is that the Retrofreak one was more designed for the asian market. Its top slot is shaped for MARK III cartridges (so raphnet adapter is no more the only way to read them with OSCR) whereas Retron one was designed for Master System carts.

I can confirm that RetroFreak RF-5 is working wih current code (v12.8) :

![image](https://github.com/sanni/cartreader/assets/87064902/23185a04-be3e-426b-8d71-0ed17769e73b)

![image](https://github.com/sanni/cartreader/assets/87064902/860fa97f-3f5f-4f0d-a4fa-79e5ac8d5d6b)

![image](https://github.com/sanni/cartreader/assets/87064902/9bd0ad93-108e-48af-83d8-b705fd71386a)

**Comment by toaks2804** [comment]
Nice! So is the card capture still required? The adapter has its own card slot doesn't it?Sent from my Galaxy
-------- Original message --------From: PsyK0p4T ***@***.***> Date: 8/16/23  3:59 PM  (GMT-08:00) To: sanni/cartreader ***@***.***> Cc: toaks2804 ***@***.***>, Author ***@***.***> Subject: Re: [sanni/cartreader] Support for Analogue / RetroFreak adapters (Discussion #693) 
Months later, I can finally answer to this !
So yes, "RetroFreak RF-5" and "Retron5 3-in-1" adapter are quite close.
A difference between them is that Retrofreak one was more designed for the asian market. Its top slot is shaped for MARK III cartridges (so raphnet adapter is no more the only way to read them with OSCR) where as Retron one was designed for Master System carts.
I can confirm that RetroFreak RF-5 is working wih current code (v12.8) :

**Reply by PsyK0p4T** [reply]
Yes there is a card slot on the adapter's left side, but for now I didn't success at dumping anything with it.
Note that SEGA cards in general are not well supported, even with the Card Catcher. And I don't have many of them. It would be interesting to do more tests with SG-1000 cards, MARK3 and European SMS ones etc.

**Comment by bedbug1226** [comment]
> ...They also have a Game Gear, Neo Geo Pocket, TG-16, and Atari Lynx that connect to the GBA port...

I would love to be able to use these as well.
I had got them for my analogue pocket but it would be nice to sue them here as well.

https://store.analogue.co/products/analogue-pocket-adapter-pack
https://store.analogue.co/products/game-gear-cartridge-adapter
